contrib/pyzfs/setup.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright 2015 ClusterHQ
```

Comment:
Did you intend to add this file?

Reply:
I thought this was fixed in a13392a ?

Reply:
No. I did not. I will rebase on crrent master to remove it.